### PR TITLE
fix(pci.instance): add the metal rescue documentation link

### PR DESCRIPTION
**Pull request by ghyenne**
| Question         | Answer
| ---------------- | ---
| Branch?          | `release/components-w8`  
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | no
| Tickets          | MANAGER-10763 
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~~Breaking change is mentioned in relevant commits~~

## Description
I added the metal rescue documentation link

## i18n
- [x] TRANS-49231

### CODE REVIEW

**Review by stif59100 — DISMISSED** [review]

File: packages/manager/modules/pci/src/projects/project/instances/instance/rescue/translations/Messages_fr_FR.json [hunk 1]
```diff
@@ -13,5 +14,8 @@
...
   "pci_projects_project_instances_instance_rescue_infos_windows": "Pour accéder à votre serveur {{instance}} en mode rescue, utilisez  :<br/><code>rdesktop {{ip}}</code> avec la clé SSH habituelle. Attention, seul le disque principal est accessible dans ce mode.",
   "pci_projects_project_instances_instance_rescue_infos_windows_with_password": "Pour accéder à votre serveur {{instance}} en mode rescue, utilisez :<br/><code>rdesktop {{ip}}</code> ou <code>ssh {{user}}@{{ip}}</code> avec le mot de passe visible dans la console VNC ou via <code>openstack console log show \"{{instance}}\"</code>. Attention, seul le disque principal est accessible dans ce mode.",
-  "pci_projects_project_instances_instance_rescue_error_rescue": "Une erreur est survenue lors du redémarrage de l'instance {{ instance }} : {{ message }}."
+  "pci_projects_project_instances_instance_rescue_error_rescue": "Une erreur est survenue lors du redémarrage de l'instance {{ instance }} : {{ message }}.",
+
```
Comment (anooparveti) [inline packages/manager/modules/pci/src/projects/project/instances/instance/rescue/translations/Messages_fr_FR.json:18]: Remove unnecessary empty line.

**Review by mohammed-zahaf — APPROVED** [review]

File: packages/manager/modules/pci/src/projects/project/instances/instance/rescue/translations/Messages_fr_FR.json (hunk 1 above)
Comment (ghyenne) [reply packages/manager/modules/pci/src/projects/project/instances/instance/rescue/translations/Messages_fr_FR.json]: It's fixed.

File: packages/manager/modules/pci/src/projects/project/instances/instance/rescue/rescue.html [hunk 2]
```diff
@@ -1,34 +1,58 @@
 <form novalidate name="rescueForm">
     <oui-modal
         data-heading="{{ 'pci_projects_project_instances_instance_rescue_title' | translate }}"
-        data-primary-action="$ctrl.rescueInstance()"
-        data-primary-label="{{:: 'pci_projects_project_instances_instance_rescue_submit_label' | translate }}"
+        data-primary-action="$ctrl.hasRescueMode ? $ctrl.rescueInstance() : $ctrl.goBack()"
```
Comment (anooparveti) [inline packages/manager/modules/pci/src/projects/project/instances/instance/rescue/rescue.html:4]: **quetsion:** If user cannot perform any action on such instance, why cant we just hide the modal?

Comment (ghyenne) [reply packages/manager/modules/pci/src/projects/project/instances/instance/rescue/rescue.html:4]: The modal is used to display the information that on the Metal instances it is different and that you have to follow the documentation.

**Review by anooparveti — APPROVED** [review]

**Review by stif59100 — APPROVED** [review]

**Review by sachinrameshn — APPROVED** [review]

**Pull request merged**